RecoBTag/TensorFlow/plugins/DeepDoubleXTFJetTagsProducer.cc

```diff
@@ -140,8 +140,9 @@ void DeepDoubleBTFJetTagsProducer::fillDescriptions(edm::ConfigurationDescriptio
     { "ID_pred/Softmax" });
   {
     edm::ParameterSetDescription psd0;
-    psd0.add<std::vector<unsigned int>>("probQ", {0});
-    psd0.add<std::vector<unsigned int>>("probH", {1});
+    psd0.add<std::vector<unsigned int>>("probQCD", {0});
+    psd0.add<std::vector<unsigned int>>("probHbb", {1});
+    psd0.addOptional<std::vector<unsigned int>>("probHcc");
```

Comment:
The `probQCD` and `probHbb` should be marked as optional as well (but keeping the default values). Otherwise, if they are missing, they will be injected in the configuration during the validation step, which, I believe, is not what you want (but I could be wrong).